RClistenTopic: "/elikos_remotecalib/input"
CommandOutputListenTopic : "/elikos_remotecalib/cmdOutListenTopic"
# RClistenTopic: "/elikos_ffmv_bottom_camera/image_raw"
RCpublishTopic: "/elikos_remotecalib/output"